.nvmrc

```
@@ -1 +1 @@
-16
+20
```

```suggestion
18
```
Let's stick with 18 for now. When v20 becomes LTS we can revisit.